ssh/defaultconfig.go

```diff
@@ -28,6 +28,9 @@ remotemoe accepts any key - see ya!`
 // DefaultConfig generates a default ssh.ServerConfig
 func DefaultConfig() (*ssh.ServerConfig, error) {
 	config := &ssh.ServerConfig{
+		BannerCallback: func(conn ssh.ConnMetadata) string {
```

Comment:
This will properly do some confusion for other people running remotemoe for them selves - lets do a environment variable instead